plugins/nodes/src/js/components/LeaderGrid.js

```diff
@@ -9,7 +9,7 @@ import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
 import ConfigurationMapSection from "#SRC/js/components/ConfigurationMapSection";
 import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
 import Loader from "#SRC/js/components/Loader";
-import { msToRelativeTime, msToDateStr } from "#SRC/js/utils/DateUtil";
```

Comment:
Shall we remove `msToDateStr` from `DateUtil` since we removed all the usages?

Reply:
That would be so cool 😻